apps/nextjs-website/react-components/components/Hero/Hero.helpers.tsx

```diff
@@ -46,11 +54,64 @@ export const HeroTextContent = ({
           textAlign={size === 'small' ? 'center' : 'left'}
         />
       </Stack>
-      {ctaButtons?.length ? (
+      {(storeButtons?.hrefGoogle || storeButtons?.hrefApple) && (
+        <Stack direction='column' spacing={1}>
+          <Typography color={textColor} fontWeight={700}>Scarica l'app</Typography>
+          <Stack
+            justifyContent='left'
+            alignItems='baseline'
+            spacing={2}
+            direction={isSmallScreen ? 'column' : 'row'}
+          >
+            {storeButtons.hrefGoogle && (
+              <Button
+                sx={{
+                  padding: '0px',
+                  marginLeft: '0px',
+                  '@media screen and (min-width: 420px)': {
+                    marginLeft: '16px',
+                  },
+                  justifyContent: 'start',
+                }}
+                key='google'
+                href={storeButtons.hrefGoogle}
+              >
+                <Image
+                  src={googleBadgeBase64}
+                  alt='Download on Google Play'
+                  height={0}
+                  width={0}
+                  style={{ height: '3em', width: 'auto' }}
+                />
+              </Button>
+            )}
+            {storeButtons.hrefApple && (
+              <Button
+                sx={{
+                  padding: '0px',
+                  margin: '0px',
+                  justifyContent: 'start',
+                }}
+                key='apple'
+                href={storeButtons.hrefApple}
+              >
+                <Image
+                  src={appleBadgeBase64}
+                  alt='Download on App store'
+                  height={0}
+                  width={0}
+                  style={{ height: '3em', width: 'auto' }}
+                />
+              </Button>
+            )}
+          </Stack>
+        </Stack>
+      )}
+      {Array.isArray(ctaButtons) && ctaButtons.length > 0 ? (
```

Comment:
```suggestion
      {ctaButtons && ctaButtons.length > 0 ? (
```
No need to check if ctaButtons is an array, just if it exists.